File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.go
func removeDuplicates(nums []int) int {
    i, j := 1, 1
    for i < len(nums) {
        if nums[i-1] != nums[i] {
            nums[j]=nums[i];
            j++;
        }
        i++
    }
    return j;
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.c
int removeDuplicates(int* nums, int numsSize) {
    int res = 0, check_number = 0;
    for (int i = 0; i < numsSize; i++) {
        if (check_number != nums[i] || i == 0) {
            check_number = nums[i];
            nums[res] = check_number;
            res++;
        }
    }
    return res;
}